kong/acl_service.go

```diff
@@ -1,5 +1,7 @@
 package kong
 
+//go:generate ifacemaker -f acl_service.go -s ACLService -i ACLServiceInterface -p kong -c "DO NOT EDIT: Auto generated" -o acl_service_interface.go
+ 
```

Comment:
This is little bit of nitpick but I think it's important.
Can we move this just above the the declaration of the service?
This helps when someone is reading only part of the code or duplicating this into another service.

This comment applies to all the generate comments.